format-proposal.md

```diff
@@ -0,0 +1,176 @@
+# Format proposal
+
+- **PRE: parameters and variables share their namespace**
+
+Parameters of any kind:
+
+```asp
+% FZN
+int : a = 1;
+% ASP
+parameter_value("a",value,1).
+
+% FZN
+float : b = 1.1;
+% ASP
+parameter_value("b",value,"1.1").
+
+% FZN
+bool : c = true;
+% ASP
+parameter_value("c",value,true).
+
+% FZN
+array [1..2] of int : d = [42,23];
+% ASP
+parameter_value("d",array,(0,value,42)).
+parameter_value("d",array,(1,value,23)).
+
+% FZN
+array [1..2] of float : e = [42.1,23.0];
+parameter_value("e",array,(0,value,"42.1")).
+parameter_value("e",array,(1,value,"23")).
+
+% FZN
+set of int: f = 23..42;
+% ASP
+parameter_value("f",range,(value,23,value,42)).
+
+% FZN
+set of float : g = {42.1,23.0};
+% ASP
+parameter_value("g",set,(value,"23"))).
+parameter_value("g",set,(value,"42.1"))).
+
+% FZN
+array [1..3] of set of int : h = [{42,17},1..5,{}];
+% ASP
+parameter_value("h",array,(0,set,(value,42))).
+parameter_value("h",array,(0,set,(value,17))).
+parameter_value("h",array,(1,range,(value,1,value,5))).
+parameter_value("h",array,(2,empty_set)).
+```
+
+Variable **assignments** of any kind.
+
+```asp
+% FZN
+var int : a = 1;
+% ASP
+variable_type("a",int).
+variable_value("a",value,1).
+
+% FZN
+var 1..3 : a;
+% ASP
+variable_type("a",range,(value,1,value,3)).
+
+% FZN
+var {1,2,3} : a;
+% ASP
+variable_type("a",set,(value,1)).
+variable_type("a",set,(value,2)).
+variable_type("a",set,(value,3)).
```

Comment:
Regarding you comment
> I still do get the output
> variable_type("X_INTRODUCED_6_",int_in_range(0,7773)).
> while int_in_range is never used in the format proposal.

I added these test cases. But I'm not sure it is correct. maybe it should be 
`variable_type("a",int,range,(value,1,value,3))).` ?